### PR TITLE
Prepare release 2.1.0

### PR DESCRIPTION
**Pull request by Shettland**
## PR checklist

- [X] Updating CHANGELOG.md and setup.py for new version 2.1.0
- [X] Make sure your code lints (`black and flake8`).
- If a new tamplate was added make sure:
    - [X] Template's schema is added in `templates/services.json`.
    - [X] Template's pipeline's documentation in `assets/reports/md/template.md` is added.
    - [X] Results Documentation in `assets/reports/results/template.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).
- [X] If you know a new user was added to the SFTP, make sure you added it to `templates/sftp_user.json`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,17 +4,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.1.0dev] - 2024-0X-0X : https://github.com/BU-ISCIII/buisciii-tools/releases/tag/2.1.X
+## [2.1.0dev] - 2024-04-19 : https://github.com/BU-ISCIII/buisciii-tools/releases/tag/2.1.0
```
Comment (Daniel-VM) [inline CHANGELOG.md:7]: Just to confirm. Should the 'dev' tag be removed in the version name when preparing for release?

**Comment by saramonzon** [comment]
Thanks for moving the "changes" to the template changes, now it's clearer!

Now, for the release, you need to move all the info for the versión 2.1.0dev below the hotfix (just before the 2.0.0), and remove the "dev" from the tag.

On the top of the document (just like we have now the 2.1.0dev) prepare a new blank space for the next release  like 2.2.0dev with every subheader blank.

File: CHANGELOG.md (hunk 1 above)
Comment (saramonzon) [reply CHANGELOG.md:7]: yes! :) I've commented about that

**Review by Daniel-VM — APPROVED** [review]
LGTM

**Review by saramonzon — APPROVED** [review]

**Pull request merged**